edx_api_doc_tools/conf_utils.py

```diff
@@ -13,12 +13,13 @@
 from rest_framework import permissions
 
 
-def make_docs_urls(api_info):
+def make_docs_urls(api_info, url_patterns=None):
```

Comment:
The name `url_patterns` on its own could be ambiguous -- it could both mean "the URLs that docs get exposed upon" and "the URLs you are including in the docs."

Consider calling this `included_url_patterns` or `api_url_patterns` or something like that.